src/BlockTypes/MiniCart.php

```diff
@@ -442,4 +442,50 @@ protected function get_tax_label() {
 	protected function get_cart_payload() {
 		return WC()->api->get_endpoint_data( '/wc/store/cart' );
 	}
+
+	/**
+	 * Prepare translations for inner blocks and dependencies.
+	 */
+	protected function get_dependencies_translations() {
+		/**
+		 * Temporary remove the this filter so $wp_scripts->print_translations
+		 * calls won't accident print the translations scripts for the block.
+		 *
+		 * $wp_scripts->print_translations() calls load_script_textdomain()
+		 * which calls load_script_translations() containing the below filter.
+		 */
+		remove_filter( 'pre_load_script_translations', 'woocommerce_blocks_get_i18n_data_json', 10, 4 );
```

Comment:
Is this really needed? I tried removing this line and line 487 and everything seemed to keep working. But maybe I'm missing something?

Reply:
Did you test it with Storefront? With the block themes, there is no issue. But with classic themes, when Mini Cart is inserted as a widget, there are some JS errors as mentioned in https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6153. I've explained the reason for this temporary removal.